src/activities/letter-in-word/LetterInWord.qml

```diff
@@ -328,7 +328,7 @@ ActivityBase {
             anchors.leftMargin: 15 * ApplicationInfo.ratio
             anchors.rightMargin: 15 * ApplicationInfo.ratio
             anchors.bottomMargin: 10 * ApplicationInfo.ratio
-            cellWidth: itemWidth + 25*ApplicationInfo.ratio
+            cellWidth: itemWidth + 20 * ApplicationInfo.ratio
             cellHeight: itemHeight + 15*ApplicationInfo.ratio
```

Comment:
15 * ApplicationInfo.ratio

Reply:
I think you mistook my comment on line 332 to 331. Spaces were to be added to "15*ApplicationInfo.ratio" on line 332 :) The cellWidth is fine as before, else it will make it too small for smaller devices.